### DISCUSSION TITLE
ctrl + shift key in notepad

### DISCUSSION THREAD

**Discussion by MrFox2681** [valinet/ExplorerPatcher#3047] [Q&A]
Hello, thank you very much for your application. I don't know if you can do this, but I have a problem with my notepad. I use ctrl shift to change the language, and while using the notepad, when switching the language (with the right ctrl shifr combination), the writing changes from right to left, can you do something so that the notepad does not react to this combination (in the settings, change the layout is " not assigned" so that's not the problem). Thanks if you answer!

**Comment by pyrates999** [comment]
For the language that you change it to, is it a left to right language or right to left like english?

Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.

Please include any other software you have installed that modifies the UI of windows.

**Comment by MrFox2681** [comment]
Now I’m not at the computer, because it’s night)(last win11 version, last
explorer patcher version). I switch Russian to English. In this case, the
writing of the text changes from the classic left to right, to right to
left, when you press the right combination (This toggles RTL and LTR
typing.), when you press the left, everything returns to normal. The
problem is with the notepad itself.

пн, 25 мар. 2024 г., 02:50 pyrates999 ***@***.***>:

**Comment by pyrates999** [comment]
Ok, please let me know when you are at your computer so you can let me know that information.

**Comment by MrFox2681** [comment]
Выпуск	Windows 11 Pro
Версия	23H2
Дата установки	‎24.‎06.‎2023
Сборка ОС	22631.3296
Взаимодействие	Windows Feature Experience Pack 1000.22687.1000.0

![изображение](https://github.com/valinet/ExplorerPatcher/assets/74142096/05257c1f-bff0-4bce-83d5-1812a1e8fe45)

**Comment by pyrates999** [comment]
Please include what settings you changed in explorer patcher properties application. These should be screen shots.

Please include any other software you have installed that modifies the UI of windows.

**Comment by MrFox2681** [comment]
![изображение](https://github.com/valinet/ExplorerPatcher/assets/74142096/14149a89-a33b-4a13-aef2-a58e1961a6b8)
![изображение](https://github.com/valinet/ExplorerPatcher/assets/74142096/8b3bdef0-640e-4e3b-b7d6-46a3cafc91c9)
![изображение](https://github.com/valinet/ExplorerPatcher/assets/74142096/1e4909ed-bf01-4fd6-893c-a197cf696753)
![изображение](https://github.com/valinet/ExplorerPatcher/assets/74142096/79210c5a-503b-4501-97dc-680260f3463f)
![изображение](https://github.com/valinet/ExplorerPatcher/assets/74142096/0fc9d232-7af7-4d06-aff4-e0cc2b44074e)

other software not install

**Comment by pyrates999** [comment]
Try changing the following in EP properties application:

1. In file explorer, try disabling **Disable the Windows 11 context menu**

Since it's in Russian, I'm guessing you enabled the above.  If it's wrong, let me know.

Perhaps you could temporarily change it to English and then paste it here?

**Comment by MrFox2681** [comment]
![изображение](https://github.com/valinet/ExplorerPatcher/assets/74142096/54d27304-e115-41e6-bd2b-c83448c6af39)
![изображение](https://github.com/valinet/ExplorerPatcher/assets/74142096/518f7355-1f5d-41be-b520-448db4cb23ea)
![изображение](https://github.com/valinet/ExplorerPatcher/assets/74142096/50215902-6573-42b6-b60b-e9a7ff62e7f9)
![изображение](https://github.com/valinet/ExplorerPatcher/assets/74142096/9e876f2e-9554-4f4b-98f9-f6f7e6a5e753)
![изображение](https://github.com/valinet/ExplorerPatcher/assets/74142096/2bef09c1-1d7a-4126-a2ec-ccab3292a3a3)
![изображение](https://github.com/valinet/ExplorerPatcher/assets/74142096/eb65c29a-2c41-4183-8f50-94ce0012591d)
![изображение](https://github.com/valinet/ExplorerPatcher/assets/74142096/e9715c34-0c0e-430a-81bf-23705038c2cf)
![изображение](https://github.com/valinet/ExplorerPatcher/assets/74142096/661c0144-5b82-43d5-b41b-7a353ae9da6a)
![изображение](https://github.com/valinet/ExplorerPatcher/assets/74142096/21ff9f58-8ae2-4333-8e4b-8ad6f896a341)

**Comment by MrFox2681** [comment]
Problem is not in EP, it's Microsoft who screwed up

**Comment by pyrates999** [comment]
Ok, thanks.  Does it still occur if you uninstall EP?

**Comment by MrFox2681** [comment]
Yes

**Comment by pyrates999** [comment]
Unfortunately this is beyond what EP is for.  But just in case, please wait for a developer to respond.
